.azure-pipelines/mgmt.yml

```diff
@@ -0,0 +1,53 @@
+# External variables Settable at queue time:
+# "pomFile" : "pom.xml".   This is the project object model file for Maven.
+
+trigger:
+  - master
+
+variables:
+  DefaultOptions: '--batch-mode -Dmaven.wagon.http.pool=false'
+  LoggingOptions: '-Dorg.slf4j.simpleLogger.defaultLogLevel=error -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn'
+
+jobs:
+  - job: 'Build'
+
+    strategy:
+      matrix:
+        Java 8:
+          ArtifactName: 'packages'
+          ProfileFlag: ''
+          JavaVersion: '1.8'
+        Java 7:
+          ArtifactName: 'packages'
+          ProfileFlag: ''
+          JavaVersion: '1.7'
+    
+    pool:
+      vmImage: 'ubuntu-16.04'
+      
+    steps:
+      - task: ShellScript@2
+        displayName: 'generate mgmt pom file'
+        inputs:
+          scriptPath: update_mgmt_pom.sh
```

Comment:
I don't like the idea of us modifying committed files as part of the build. If you want to generate this file as part of the build to help with wildcarding then lets create a temporary file and not overwrite the pom.mgmt.xml file and we can give a warning in the build if the 2 files are not identical.

Reply:
not doin it anymore... figured out another way to do this